src/controllers/newsController.js

```diff
@@ -0,0 +1,44 @@
+const Article = require('../models/article');
+
+function getAllNews(req, res, next) {
+  Article.find()
+    .then(articles => {res.send(articles)});
+}
+
+function postNewNewsItem(req, res, next) {
+  Article.create(req.body)
+    .then(() => res.json({ message: 'The news item was added' }))
+    .catch(() => res.status(422).send({ error: `There is no title value in request body!` }));
```

Comment:
The validation error should come from initial place.
This function is not aware of 'Title' and article scheme